R/shared-methods.R

```diff
@@ -134,13 +137,13 @@ DataClass <- R6::R6Class(
     supported_region_names = list("1" = NA),
     #' @field supported_region_codes A list of region codes in order of level.
     supported_region_codes = list("1" = NA),
-    #' @field region_name string Name for the codes column, e.g. 'iso_3166_2'
+    #' @field region_name string Name for the region column, e.g. 'region'
     region_name = NULL,
     #' @field code_name string Name for the codes column, e.g. 'iso_3166_2'
     code_name = NULL,
     #' @field codes_lookup string or tibble Region codes for the target country
```

Comment:
as @RichardMN  mentioned good to point out here and elsewhere that this is a field that will get filled out at initialisation.